modules/sdk-core/src/bitgo/utils/tss/ecdsa/ecdsaMPCv2.ts

```diff
@@ -368,6 +369,50 @@ export class EcdsaMPCv2Utils extends BaseEcdsaUtils {
     return { ...(await keychains.add(recipientKeychainParams)), reducedEncryptedPrv: reducedEncryptedPrv };
   }
 
+  async getMpcV2RetrofitDataFromMpcV1Keys(params: {
+    mpcv1UserKeyShare: string;
+    mpcv1BackupKeyShare: string;
+    bitgoBigSi: string;
```

Comment:
is this going to be the string that we are going to get from the backend part of the retrofit flow that mohammad will work on?

Reply:
yes

Reply:
It would help if we create some codecs that the WP and SDK can use to validate these after parsing into a json

Reply:
this would never be used by WP so not sure a codec would be useful here

Reply:
Well, whatever the input is, can be passed through a codec to check if its correct, right now you are JSON parsing a string and hopping that is correct.
Same for whatever data goes to WP, it can be defined a in a codec so WP can verify and used here as an output type

Reply:
since this is a migration feature i dont think it's worthwhile creating a codec for this since there are assertions after the parsing that check some of the fields.